xonsh/xoreutils/__init__.py

```diff
@@ -0,0 +1,6 @@
+from xonsh.xoreutils.echo import echo
+from xonsh.xoreutils.pwd import pwd
+from xonsh.xoreutils.cat import cat
+from xonsh.xoreutils.tty import tty
+from xonsh.xoreutils.tee import tee
+from xonsh.xoreutils.yes import yes
```

Comment:
ah, i wasn't aware of that. `flake8` is relaxed there.

interesting read: https://www.python.org/dev/peps/pep-0328/